02-dz0/01-semantic-mockup/solution.html

```diff
@@ -1,4 +1,82 @@
 <!DOCTYPE html>
 <html lang="ru">
     <!-- Ваше решение -->
+    <head>
+        <meta charset="utf-8">
+        <style>
+            html{
+                color: white;
+                text-align: center;
+                line-height: 3rem;
+                font-size: 20px;
+            }
+            html > *{
```

Comment:
На самом деле, это тоже самое, что и
```suggestion
            * {
```

а селектор сложнее. Давай поправим?)

Reply:
я использовал данный селектор чтобы не сбросить отступы списка.

Reply:
окей, поняла